src/v0/destinations/impact/util.js

```diff
@@ -78,6 +78,7 @@ const populateProductProperties = (productsMapping, properties) => {
   const productProperties = {};
```

Comment:
Need to have unit test for entire `populateProductProperties` function too - logic seems too convoluted

Reply:
Added the util test cases. Please Check